javascript-timers/interval.js

```diff
@@ -0,0 +1,11 @@
+var $countdownText = document.querySelector('.countdown-display');
+var intervalID = setInterval(countdown, 1000);
+
+function countdown() {
+  if ($countdownText.textContent === '0') {
+    clearInterval(intervalID);
+    $countdownText.textContent = '~Earth Beeeeellllooowwwww Us~';
+  } else {
+    $countdownText.textContent -= intervalID;
```

Comment:
What if the `intervalId` was anything other than 1? This would break colossally 😬